src/components/measureLanding/NewMeasure.test.tsx

```diff
@@ -61,8 +61,8 @@ describe("Measure Page", () => {
     });
   });
 
-  test("shows all measures measures on tab click", async () => {
-    act(async () => {
+  test.skip("shows all measures measures on tab click", async () => {
+    await act(async () => {
```

Comment:
1. Are we anticipating to not skip them in future ? If so I would like to see a comment on why these tests are being skipped.
2. Is it possible to move NewMeasure.tsx to its own directory? It looks like component and its associated test case are in different places.